### ISSUE TITLE
[telemelody] Too many errors of template2melody when generating training data

### ISSUE THREAD

**Issue by gandolfxu** [microsoft/muzic#98]
Does it conforms to expectation?

![Image_20230118112426](https://user-images.githubusercontent.com/50284410/213076835-5545e323-6ade-4f45-bb77-a28023677697.png)

**Comment by jzq2000** [comment]
Yes, it conforms to expectation. The processing filters out non 4/4, duplicate and blank midis, so it is meeting the expected standards.

**Comment by gandolfxu** [comment]
train: 21332/108590 (19.64%) midi files successfully processed
valid: 93/1109 (8.39%) midi files successfully processed
test: 74/1114 (6.64%) midi files successfully processed

I find that the number of midi data for training is 21332, which is different from that in the paper. In the paper, this number is 45129. @ @jzq2000

**Comment by jzq2000** [comment]
Hi, 45129 is the total number of midi samples in [LMD-matched dataset](https://colinraffel.com/projects/lmd/).  Maybe you can check whether there are duplicate/other midis under your data_dir.
